stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2564,23 +2590,40 @@ final internal class _Native${Self}StorageImpl<${TypeParameters}> :
   }
 %end
 
-  /// Create a storage instance with room for 'capacity' entries and all entries
-  /// marked invalid.
-  internal class func create(capacity: Int) -> StorageImpl {
+  internal static func create(minimumCapacity: Int) -> Storage {
+    // Make sure there's a representable power of 2 >= minimumCapacity
+    _sanityCheck(minimumCapacity <= (Int.max >> 1) + 1)
+    var capacity = 2
+    while capacity < minimumCapacity {
+      capacity <<= 1
+    }
+    return create(capacity: capacity)
+  }
+
+  /// Create a storage instance with room for at least 'minimumCapacity'
+  /// entries and all entries marked invalid.
+  internal static func create(capacity: Int) -> Storage {
+    defer { _fixLifetime(self) }
```

Comment:
You don't need this `_fixLifetime`, it is a static function (and `self` is the class object).